examples/carousel/esp-idf/s2-kaluga-kit/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ cmake_minimum_required(VERSION 3.14)
 
 include($ENV{IDF_PATH}/tools/cmake/project.cmake)
 
-set(SLINT_ESP_LOCAL_EXAMPLE ON)
+set(SLINT_ESP_LOCAL_EXAMPLE ON CACHE BOOL "Build example with local version of Slint")
```

Comment:
It is strange to define it as a CACHE variable here and in every example. But I understand why it is like that.